### ISSUE TITLE
[Question] Is it possible to display a defined character when entering sensitive value for options?

### ISSUE THREAD

**Issue by shaggygi** [dotnet/command-line-api#1645]
Is there support to allow defining a character to be displayed when entering values for options?  For example, I have a password options and would like to show '*' when typing into the console.

Sorry if I overlooked, but didn't see anything in docs.

**Comment by elgonzo** [comment]
Not really, because what you are asking for would have to be implemented by the console/shell itself (and possibly related OS functions that allow access to/inspection of the commandlines of running programs/processes).
